### PR TITLE
Create pifinder.rb

### PR DESCRIPTION
**Pull request by thibmaek**
##### Instructions
- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
#### Adding a new cask
- [ ] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [ ] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [ ] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Commit message includes cask’s name.
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
### Changes to the core
- [ ] Followed [hacking.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/hacking.md).

### CODE REVIEW

**Comment by jellyfishcoder** [comment]
@thibmaek Delete the instructions section, editing an existing cask section, and changes to the core section. If you did the things in it check the boxes.

**Comment by vitorgalvao** [comment]
You’re not editing an existing cask, you’re adding a new one. I don’t see what the confusion is there, but if it is somehow unclear, please tell us why so we can fix it.

Also, you need to do the requested actions. This submission is now failing precisely because they were not run (and fixed).

File: Casks/pifinder.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'pifinder' do
+  version '3.0.0'
+  sha256 'b6aab9b73d50dd0652af5ecc9cc841bc050f4d7a457b9a11ec623d0c09a338d6'
+
+  # github.com/adafruit/Adafruit-Pi-Finder was verified as official when first introduced to the cask
+  url "https://github.com/fitztrev/shuttle/releases/download/v#{version}/Shuttle.zip"
+  url "https://github.com/adafruit/Adafruit-Pi-Finder/releases/download/#{version}/PiFinder-#{version}-osx-x64.zip"
```
Comment (vitorgalvao) [inline Casks/pifinder.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by vitorgalvao** [comment]
There are a few errors here, like the double `url`s. I ask that you try again in a new PR.

**Pull request closed without merging**